versioned_docs/version-3.0.0-beta/organization/personal-accounts.md

```diff
@@ -0,0 +1,28 @@
+---
+id: personal accounts
+title: Personal Accounts
+sidebar_label: Personal Accounts
+slug: /organization/personal-accounts
+---
+
+## Personal Accounts
+
+When you create a new account with Spheron, you are creating a Personal Account. A Personal Account is your own, individual account, registered with the Spheron platform and can be upgraded and can act as an admin account for your organization. 
+
+## Understanding the Personal Account and Pro Plan 
+Your Personal Account always exists on the Starter plan and is ideal for personal projects. While you can collaborate with others via pull requests on your repository, you cannot invite others to collaborate on a Team project with a Starter Plan Account. 
+
+<img src={require('@site/static/img/3.0.0/personalaccounts-organization.png').default} />
+
+## Upgrading to a pro plan 
+
+Choosing to upgrade to a Pro plan and enable you to collaborate on Projects. When you create a Team, you will want to transfer any Projects you wish to collaborate on from your Personal Account to the Team.
+
+<img src={require('@site/static/img/3.0.0/personalaccounts-plan.png').default} />
+
+## Login methods
+You can log into your Personal Account using the following methods:
+
+- GitHub
+- GitLab
+- Bitbucket
```

Comment:
Add this too

For more information on how to login to your Personal Account, please see [Login Connections].

Reply:
Made a new section for it and added login connections